bundles/org.openhab.binding.homekit/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="homekit"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="accessory">
+		<label>HomeKit Accessory</label>
+		<description>HomeKit accessory with its own LAN connection</description>
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>IP Address</label>
+				<description>IP v4 address of the HomeKit accessory.</description>
```

Comment:
The name or label doesn't mention that, so it would be good if the description did:
```suggestion
				<description>IP v4 address and port of the HomeKit accessory separated by colon.</description>
```

Or even better: Keep them in separate parameters, `ipAddress` and `port` to make it simpler to set and parse. 😉

Reply:
I won't add an extra parameter for the port, since blank and 0 port values are allowed, and also port 80 is considered standard (and thus dropped).